packages/react/src/verbs/Ungroup/Ungroup.tsx

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import React, { memo } from 'react'
+import { LeftAlignedRow, VerbContainer } from '../../common'
+import { StepComponentProps } from '../../types'
+
+/**
+ * Provides inputs for an ungroup step.
+ */
+export const Ungroup: React.FC<StepComponentProps> = memo(function Ungroup({
```

Comment:
Ungroup and Unroll could probably both be collapsed into something like a "NoParameters" component that just returns null. This is like the SetOperation - each set verb (difference, intersect, etc.) returns the same SetOperation component in selectstepComponent.